ch_backup/logic/table.py

```diff
@@ -654,20 +648,33 @@ def _restore_data(
         for table_meta in tables:
             cloud_storage_parts = []
             try:
+                maybe_table_short = context.ch_ctl.get_table(
+                    table_meta.database, table_meta.name, short_query=True
+                )
+                assert (
```

Comment:
Let's use `if` and an exception instead of `assert`. `maybe_table_short` can be `None`, but asserts can be disabled in runtime